src/test/java/reposense/util/FileUtilTest.java

```diff
@@ -5,22 +5,48 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashSet;
 
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Test;
 
+import reposense.report.SummaryReportJson;
+
 public class FileUtilTest {
 
     private static final Path FILE_UTIL_TEST_DIRECTORY = new File(FileUtilTest.class.getClassLoader()
             .getResource("FileUtilTest").getFile()).toPath().toAbsolutePath();
+    private static final Path RELEVANT_REPO_DIRECTORY_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
+            "reposense-report-test");
     private static final Path ARCHIVE_ZIP_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
             FileUtil.ZIP_FILE);
     private static final Path EXPECTED_UNZIPPED_DIRECTORY_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
             "expectedUnzip");
     private static final Path TEST_ZIP_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(), "testZip.zip");
     private static final Path UNZIPPED_DIRECTORY_PATH = Paths.get(FILE_UTIL_TEST_DIRECTORY.toString(),
             "UnzippedFolder");
+    private static final Path[] RELEVANT_FILE_FOLDER_PATHS = {
+            Paths.get(RELEVANT_REPO_DIRECTORY_PATH.toString(), "reposense_testrepo-Beta").toAbsolutePath(),
+            Paths.get(RELEVANT_REPO_DIRECTORY_PATH.toString(), "reposense_testrepo-Charlie").toAbsolutePath(),
+            Paths.get(RELEVANT_REPO_DIRECTORY_PATH.toString(), SummaryReportJson.SUMMARY_JSON_FILE_NAME)
+                    .toAbsolutePath()};
```

Comment:
The line break here. In this case, break after the `,` is better than before `.`.